packages/shared/src/components/dropdown/DropdownMenu.tsx

```diff
@@ -111,7 +111,9 @@ export const DropdownMenuContent = React.forwardRef<
         className={classNames('DropdownMenuContent', className)}
         align={align}
       >
-        {children}
+        <div className='max-h-60 overflow-y-auto bg-inherit'>
```

Comment:
One final request from design team.
They want it to be 280px

Could you verify if `max-h-70` works or if we have to use `max-h-[17.5rem]`

@sshanzel after that if you happy you can merge.